phase0/CRC Design/Player.txt

```diff
@@ -1,6 +1,7 @@
-Class name: entities.Player
+Class name: Player
 Parent class: 
-Subclasses: 
+Subclasses:
+Architecture Section: Entity
```

Comment:
Think this would be more of a controller as it essentially is managing the game rn